### ISSUE TITLE
Nests not always in parks

### ISSUE THREAD

**Issue by Coraz97** [Tr4sHCr4fT/nestmap#2]
This one (43.792224, 11.250505) is an Electabuzz nest, but bootstrap.py doesn't get it because it's not in a green zone (I think).

**Comment by tcmaps** [comment]
Yes, true. PoGo spawns were created by map tags, like this for 'park'  
Nestmap can't 'see' that data... would need some ext. api, like OSM
